mkdocs.yml

```
@@ -132,7 +136,10 @@ plugins:
       redirect_maps:
         "01-introduction.md": "introduction.md"
         "02-common-principles.md": "common-principles.md"
-        "03-modality-agnostic-files.md": "modality-agnostic-files.md"
+        "03-modality-agnostic-files/01-dataset-description.md": "modality-agnostic-files/dataset-description.md"
+        "03-modality-agnostic-files/02-data-summary-files.md": "modality-agnostic-files/data-summary-files.md"
+        "03-modality-agnostic-files/03-phenotypic-and-assessment-data.md": "modality-agnostic-files/phenotypic-and-assessment-data.md"
+        "03-modality-agnostic-files/04-code.md": "modality-agnostic-files/code.md"
```

This breaks old links that point to `03-modality-agnostic-files.html`, while these new files never existed, so don't need redirects. I think we just want:

```suggestion
        "03-modality-agnostic-files.md": "modality-agnostic-files/dataset-description.md"
```

Once https://github.com/mkdocs/mkdocs-redirects/pull/73 is released, we could redirect the headings to the appropriate files.